x/posts/internal/keeper/keeper.go

```diff
@@ -41,6 +42,18 @@ func (k Keeper) GetLastPostID(ctx sdk.Context) types.PostID {
 	return id
 }
 
+func (k Keeper) GetPostHashtags(post types.Post) []string {
+	re := regexp.MustCompile(`(?:|^)#[A-Za-z0-9]+(?:|$)`)
+	hashtags := re.FindAllStringSubmatch(post.Message, -1)
+
+	var hts = []string{}
+	for _, hashtagSlice := range hashtags {
+		hts = append(hts, hashtagSlice[0])
+	}
```

Comment:
I think that this `for` loop might create a slice having duplicated items inside if the message contains the same hashtag multiple times. For example, considering the following message: 

_This is a #test message which has multiple #test hashtags inside_

Calling `GetPostHashtags` will return `[test, test]` instead of `[test]`.

Can we fix this? Also, it may be a good idea to add a test case like this where a message has multiple times the same hashtag inside.